scripts/validate-build.js

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env node
+// eslint-disable-next-line strict
+'use strict';
+
+const fs = require('fs');
+const esprima = require('esprima');
+const walk = require('esprima-walk');
+const colors = require('colors');
+
+/**
+ * Load the Abstract Syntax Tree from the javascript file given
+ *
+ * @param {String} file - The path to the javascript filename
+ * @returns {ASTTree} Returns the AST tree
+ */
+function loadAST(file) {
+  try {
+    return esprima.parse(
+      String(fs.readFileSync(file)),
+      { comment: true }
+    );
+
+  } catch (e) {
+    console.error('Unable to parse source:', e);
+
+    return false;
+  }
+}
+
+/**
+ * Collect all comments from the AST
+ *
+ * @param {ASTTree} ast - The AST tree
+ * @returns {Array<ASTNode>} Returns the comment AST nodes
+ */
+function loadComments(ast) {
+  let comments = [];
+
+  walk(ast, function (node) {
```

Comment:
You are right... for some reason the `comment` property was not appearing on the AST on my tests, but now that I re-tested it on a fresh installation it seems to work... I really don't know why 😄 

Thanks for this 👍